### PR TITLE
Added The Zen of Python

### PR DESCRIPTION
**Pull request by gauravsitlani**

### CODE REVIEW

File: file_and_modules.md [hunk 1]
```diff
@@ -41,7 +41,35 @@
 * We can import a module using import keyword.
 * If it is not available it will give import error.
 * dir() method tells what these modules provides.
+* What's [The Zen of Python] (https://www.python.org/dev/peps/pep-0020/) ? 
 ```
+ # python
+Python 3.6.8 (default, Jan 31 2019, 09:38:34) 
+[GCC 8.2.1 20181215 (Red Hat 8.2.1-6)] on linux
+Type "help", "copyright", "credits" or "license" for more information.
```
Comment (chkumar246) [inline file_and_modules.md]: Remove this and just use python interpreter, otherwise it will confuse user to use older version.

Comment (gauravsitlani) [reply file_and_modules.md]: done!

**Review by chkumar246 — APPROVED** [review]
Thanks for adding this, @bhavin192 please have a look.

File: file_and_modules.md [hunk 2]
```diff
@@ -41,7 +41,31 @@
 * We can import a module using `import` keyword.
 * If it is not available it will give import error.
 * dir() method tells what the attribute provides.
+* What's [The Zen of Python] (https://www.python.org/dev/peps/pep-0020/) ?
```
Comment (bhavin192) [inline file_and_modules.md]: * What's [The Zen of Python] (https://www.python.org/dev/peps/pep-0020/) ? 

It's getting rendered like this, can you please fix it?

**Review by bhavin192 — CHANGES_REQUESTED** [review]

**Comment by chkumar246** [comment]
@digitronik Please have a look, Thanks :-)

**Comment by bhavin192** [comment]
@gauravsitlani can you please squash the commits once done?

File: file_and_modules.md (hunk 2 above)
Comment (digitronik) [reply file_and_modules.md]: @gauravsitlani yup... please remove `white-space`.

**Review by digitronik — COMMENTED** [review]
Only `url` rendering problem. else LGTM :+1:

**Review by bhavin192 — APPROVED** [review]

**Pull request merged**